pkg/components/aws-ebs-csi-driver/component.go

```diff
@@ -60,7 +62,7 @@ func (c *component) RenderManifests() (map[string]string, error) {
 		return nil, fmt.Errorf("loading chart from assets failed: %w", err)
 	}
 
-	values, err := util.RenderTemplate(chartValuesTmpl, c)
+	values, err := template.Render(chartValuesTmpl, c)
```

Comment:
Side note: the naming is so confusing and easily overlooked even in future.

Reply:
What is confusing in this naming? we **Render** the **template**. How would you change it to improve it?

Reply:
I meant `util.RenderTemplate` vs `template.Render` is confusing.

Reply:
Ignore it, I was confused.